races/chicken.md

```diff
@@ -1,3 +1,45 @@
 ---
-wrapper: "../../../js/components/wiki/StubArticle"
+wrapper: "../../../js/components/wiki/BasicArticle"
```

Comment:
I changed the way wrapper and prependJs work.
You should now use the wikiWrapper and muiComponents settings. See the Wiki index page for more info.

(I see your PR also now has merge conflicts because I updated all the existing articles on master to use the new system.)